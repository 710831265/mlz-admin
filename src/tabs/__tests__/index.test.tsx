import React, { Component } from 'react';
import testMount from '../../../tests/testMount';
import testSnapshot from '../../../tests/testSnapshot';
import Switch from '..';

describe('ðŸ§ª Switch', () => {
  testMount(Switch);
  testSnapshot(Switch);
});
