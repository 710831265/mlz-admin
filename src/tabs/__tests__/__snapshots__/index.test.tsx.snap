// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Switch snapshot test snapshot comparing 1`] = `"<div class=\\"ant-tabs ant-tabs-top\\"><div role=\\"tablist\\" class=\\"ant-tabs-nav\\"><div class=\\"ant-tabs-nav-wrap\\"><div class=\\"ant-tabs-nav-list\\" style=\\"transform: translate(0px, 0px);\\"><div class=\\"ant-tabs-ink-bar ant-tabs-ink-bar-animated\\"></div></div></div><div class=\\"ant-tabs-nav-operations ant-tabs-nav-operations-hidden\\"><button type=\\"button\\" class=\\"ant-tabs-nav-more\\" style=\\"visibility: hidden; order: 1;\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" aria-haspopup=\\"listbox\\" aria-controls=\\"rc-tabs-test-more-popup\\" id=\\"rc-tabs-test-more\\" aria-expanded=\\"false\\"><span role=\\"img\\" aria-label=\\"ellipsis\\" class=\\"anticon anticon-ellipsis\\"><svg viewBox=\\"64 64 896 896\\" focusable=\\"false\\" class=\\"\\" data-icon=\\"ellipsis\\" width=\\"1em\\" height=\\"1em\\" fill=\\"currentColor\\" aria-hidden=\\"true\\"><path d=\\"M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z\\"/></svg></span></button></div></div><div class=\\"ant-tabs-content-holder\\"><div class=\\"ant-tabs-content ant-tabs-content-top\\"></div></div></div>"`;
